src/main/java/gregtech/common/CommonProxy.java

```diff
@@ -321,6 +322,10 @@ public static void registerRecipesLowest(RegistryEvent.Register<IRecipe> event)
         if (GroovyScriptCompat.isLoaded()) {
             GroovyScriptCompat.loadMetaItemBracketHandler();
         }
+        // On initial load we need to postpone cache flushing until FMLPostInitializationEvent
+        // to account for WoodMachineRecipes#postInit().
+        if (Loader.instance().hasReachedState(LoaderState.AVAILABLE))
+            GTRecipeInputCache.disableCache();
```

Comment:
With other changes made to `WoodMachineRecipes` (now named `WoodRecipeLoader`), can this be moved into an earlier phase now?

Also alternatively, would it be better to only flush the cache after CraftTweaker scripts run too?

Reply:
I left cache flushing to be delayed until post-load because addons might register their recipes after registry event, similar to previous iteration of `WoodMachineRecipes`. I figured it might be useful for some.
I don't know when does CT script register recipes so you'd have to tell me what's wrong 😛